component.js

```diff
@@ -102,6 +103,7 @@ function factory (options) {
   var _isJsx = !!options.jsx;
   var _hiddenCursorField = options.cursorField || '__singleCursor';
   var _isNode = options.isNode || isNode;
+  var _cached = cached.withDefaults(_shouldComponentUpdate);
```

Comment:
Now you do 

```js
function factory (methods) {
  var isEqual = (methods && methods.isEqualProps) || shouldupdate.isEqualProps;
  // ...
```

And passing in the function like:

```js
var _cached = cached.withDefaults(_shouldComponentUpdate);
```

Wouldn't work. You'd always use `shouldupdate.isEqualProps`. But passing in `options` instead, would work.

yeah, `isEqualProps` would work! :)